.travis.yml

```diff
@@ -4,11 +4,9 @@ php:
   - 5.6
   - 7.0
   - 7.1
-  - nightly
```

Comment:
You should not delete `nightly` version. It is not `7.2`, it is current `dev` version, so now it is `7.3`. We always should know how our code running on new version of PHP

Reply:
I disagree. 7.3 is now very far from being usable, and Travis will tag it while in beta stage, so we can add it at that point. (which will be in at least 8-9 months in the future from now).